### PR TITLE
Autotests: #3318 - Create autotests for attachment point tool

### PR DESCRIPTION
**Pull request by MartaWilliams**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

**Review by chgayane — APPROVED** [review]

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  receiveFileComparisonData,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+test.describe('Attachment Point Tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  // test.afterEach(async ({ page }) => {
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts:18]: Could you remove this comment or uncomment this line?

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  //   await takeEditorScreenshot(page);
+  // });
+
+  test('Opening *.mol V2000 file with S-Group containing Attachment point', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-18024
+    await openFileAndAddToCanvas('s-group-with-attachment-points.mol', page);
+    // check that structure opened from file is displayed correctly
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts:27]: You can remove this comment it is pretty self-explanatory

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    await saveToFile('KET/test-EPMLSOPKET-18025.ket', expectedFile);
+
+    const { fileExpected: ketFileExpected, file: ketFile } =
+      await receiveFileComparisonData({
+        page,
+        expectedFileName: 'tests/test-data/KET/test-EPMLSOPKET-18025.ket',
+      });
+    expect(ketFile).toEqual(ketFileExpected);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await openFileAndAddToCanvas('KET/test-EPMLSOPKET-18025.ket', page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/actions-with-attachment-point-tool.spec.ts:58]: Can you give more meaningful names for files? Which structure is stored in the file?

**Review by Nitvex — CHANGES_REQUESTED** [review]

**Review by chgayane — APPROVED** [review]

**Review by ViktoriaTkacheva — APPROVED** [review]

File: ketcher-autotests/tests/test-data/Molfiles-V2000/test-EPMLSOPKET-18025.mol [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (Nitvex) [inline ketcher-autotests/tests/test-data/Molfiles-V2000/test-EPMLSOPKET-18025.mol:1]: This file doesn't seem to be used, can you remove it?

**Review by Nitvex — APPROVED** [review]

**Pull request merged**